cellpack/autopack/Analysis.py

```diff
@@ -475,7 +475,6 @@ def rectangle_circle_area(self, bbox, center, radius):
                 area = self.g.get_rectangle_cercle_area(
```

Comment:
main change here is generalizing the 'show plotly plot' command line option to show the grid spheres if the packing is in 3d